README.md

```diff
@@ -17,46 +30,87 @@ minikube image load ipa:current
 
 The image tag needs to match Helm's `values.yaml`.
 
-### Configuration
+### Keys and configuration
 
 You will need a set of keys, certificates and a `network.toml` to start each helper. These files are expected to be placed in the `config` folder.
 
-After you built IPA (not IPA infra, this project):
+The following binaries require you to compile the IPA project (not ipa-infra). Please change the values accordingly, using `localhost` if you intend to test locally.
+
 ```
 cd <IPA project>/target/release/
 mkdir config
 
 # generate keys and cert
-./helper keygen --name localhost --tls-cert config/h1.pem --tls-key config/h1-cert.key --mk-public-key config/h1_mk.pub --mk-private-key config/h1_mk.key
-./helper keygen --name localhost --tls-cert config/h2.pem --tls-key config/h2-cert.key --mk-public-key config/h2_mk.pub --mk-private-key config/h2_mk.key
-./helper keygen --name localhost --tls-cert config/h3.pem --tls-key config/h3-cert.key --mk-public-key config/h3_mk.pub --mk-private-key config/h3_mk.key
+./helper keygen --name helper1.prod.ipa-helper.dev --tls-cert config/h1.pem --tls-key config/h1-cert.key --mk-public-key config/h1_mk.pub --mk-private-key config/h1_mk.key
```

Comment:
might be worth using a placeholder? this is a domain Meta owns, but these are public docs. granted, I doubt folks are looking at them, but the intended audience is a generic developer, right?